jsonschema2pojo-cli/pom.xml

```diff
@@ -50,8 +50,8 @@
                 <configuration>
                     <archive>
                         <manifest>
-                            <classpathPrefix>./</classpathPrefix>
                             <addClasspath>true</addClasspath>
+                            <classpathPrefix>lib/</classpathPrefix>
```

Comment:
What's the reason for this change?

Reply:
This was so that the jar can be run in the target directory and it can locate the jar files in the lib folder.

Reply:
This will break our releases so I think we need to remove it.

Reply:
Fixed